client/src/components/TrackGroup/TrackGroup.tsx

```diff
@@ -76,38 +76,62 @@ const ImageAndDetailsWrapper = styled.div`
   }
 `;
 
-const AboutWrapper = styled.div`
-  max-width: 70%;
+const AboutWrapper = styled.div<{
+  trackGroupCredits: string;
```

Comment:
Did TypeScript let this pass? Here you have `AboutWrapper` have only one prop, `trackGroupCredits` But on line 280 you pass it `trackGroupAbout`.

Reply:
No there was an error I missed it sorry